src/lib/modals/Modal.svelte

```diff
@@ -114,17 +107,23 @@
 </script>
 
 {#if open}
+  <!-- backdrop -->
+  <div class={classNames('fixed inset-0 z-40', backdropClasses)} />
+  <!-- dialog -->
```

Comment:
If I understand this right.. we added here a new \`div\` which represents the backdrop..